test/server_test.py

```diff
@@ -235,6 +254,23 @@ def started_imageio(tmpdir, drop_privileges="true"):
         proc.wait()
 
 
+@requires_root
+def test_load_ticket(tmpdir, tmp_dirs, tmp_image, ticket):
+    data = b"a" * 512 + b"b" * 512
+    tmp_image.write(data)
+    with started_imageio(tmpdir, drop_privileges="false", ticket=ticket):
```

Comment:
why do we need to use drop_privileges="false" for this test? this feature should not depend on using initial ticket.

Reply:
Was failing in the CIs (and locally if run as root). This made it pass. But I am still doing changes with the test.